### ISSUE TITLE
[?BUG] Deekseek-v2-lite checkpoint convert problem

### ISSUE THREAD

**Issue by xiaoxin83121** [alibaba/Pai-Megatron-Patch#451]
Hi, devs, thanks a lot for your contribution. I have run successfully  following the [readme](https://github.com/alibaba/Pai-Megatron-Patch/tree/main/examples/deepseek_v2). However, I found the output of the converted huggingface-format checkpoint is consist of "\n" or "[". So, I tried to only convert the official deepseek-v2-lite-chat(hf) to mcore and then convert it to hf again without training. BUT the results seem still weird, not even a sentence. Is there any bug when converting?

reproduce scripts:
``` 
- hf2mcore
export MP_PP0_LAYERS=6
cd Pai-Megatron-Patch/toolkits/model_checkpoints_convertor/deepseek
bash hf2mcore_deepseek_v2_moe_convertor.sh \
A2.4B \
path/DeepSeek-V2-Lite-Chat \
path/deepseek-ckpts/DeepSeek-V2-Lite-Chat-to-mcore-tp1-pp4-ep2  \
1  \
4  \
2 \
bf16 \
false ```  


- mcore2hf
export MP_PP0_LAYERS=6
cd Pai-Megatron-Patch/toolkits/model_checkpoints_convertor/deepseek
bash hf2mcore_deepseek_v2_moe_convertor.sh \
A2.4B \
path/deepseek-ckpts/DeepSeek-V2-Lite-Chat-to-mcore-tp1-pp4-ep2  \
path/deepseek-ckpts/DeepSeek-V2-Lite-Chat-mcore-test-to-hf \
1  \
4  \
2 \
bf16 \
true \
path//DeepSeek-V2-Lite-Chat
```

infer scripts are the same as the official deepseek-v2: [url](https://huggingface.co/deepseek-ai/DeepSeek-V2-Lite-Chat#chat-completion)

I got `"}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_}'_"` when I ask `我们如何在日常生活中减少用水？`.  And some clue might be useful for you when I convert to mcore, it shows   
`layer:26, kv_a_norm_in, diff: 5106, diff>1e-05:[2556/5120] diff_max:5.15625
layer:26, kv_a_norm_out, diff: 5104, diff>1e-05:[2550/5120] diff_max:5.15625
layer:26, kv_a_norm_out_weight, diff: 0, diff>1e-05:[0/512] diff_max:0.0
layer:26, kv_b_proj_in, diff: 5104, diff>1e-05:[2550/5120] diff_max:5.15625
layer:26, kv_b_proj_out, diff: 40632, diff>1e-05:[20123/40960] diff_max:4.15625
layer:26, kv_b_proj_out_weight, diff: 0, diff>1e-05:[0/2097152] diff_max:0.0
layer:26, o_proj_in, diff: 20312, diff>1e-05:[9890/20480] diff_max:2.296875
layer:26, o_proj_out, diff: 20304, diff>1e-05:[10037/20480] diff_max:6.5625
layer:26, o_proj_out_weight, diff: 0, diff>1e-05:[0/4194304] diff_max:0.0
layer:26, attn_out, diff: 20304, diff>1e-05:[10037/20480] diff_max:6.5625
layer:26, shared_experts_down_proj_in, diff: 28113, diff>1e-05:[14046/28160] diff_max:41.0
layer:26, shared_experts_down_proj_out, diff: 20357, diff>1e-05:[10011/20480] diff_max:94.0
layer:26, shared_experts_down_proj_out_weight, diff: 0, diff>1e-05:[0/5767168] diff_max:0.0
layer:26, lmhead, diff: 1017903, diff>1e-05:[674078/1024000] diff_max:19.875
layer:26, lmhead_weight, diff: 0, diff>1e-05:[0/209715200] diff_max:0.0
layer:26, lmhead_token, diff: 6, diff>1e-05:[4/10] diff_max:26911
logits: 1017903, diff>1e-05:[674078/1024000] diff_max:19.82421875`

Is it a bug or I just set something falsely (I have tried change bf16 to fp32, does not work). Hope to get response

**Comment by xiaoxin83121** [comment]
Problem solved. Many thanks for your swift response and dedication to the open-source community.
